src/Symfony/Component/Security/Core/Event/VoteEvent.php

```diff
@@ -46,7 +51,7 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
-    public function getVote(): int
+    public function getVote(): VoteInterface
```

Comment:
I think this should be BC no?

Reply:
Yes it can, but now I hesitate, 
make a getDecision as other classes or put an argument $asObject with default value to false to get int, true : get VoteObject, if with an argument, it can be good to make the same thing with others and delete all getDecision functions.
I'm thinking on it